src/RestAPI.php

```diff
@@ -38,8 +43,9 @@ public function __construct($secret_key, $publishable_key, $env)
 
         $this->session = new Session($this);
         $this->customer = new Customer($this);
+        $this->refund = new Refund($this);
         $this->payment_methods = new PaymentMethods($this);
-
+        $this->payment  = new Payment($this);
         // new ThawaniAjax($this);
```

Comment:
All looks good to me, just this commented code.

Reply:
This was from before, just removed the line